src/styles.css

```
@@ -46,13 +85,16 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 /* ── THEME TOGGLE ── */
 .theme-toggle{display:inline-flex;align-items:center;gap:6px;padding:5px 9px;border-radius:8px;border:1.5px solid var(--border);background:transparent;cursor:pointer;font-family:'Plus Jakarta Sans',sans-serif;transition:all .18s;flex-shrink:0}
 .theme-toggle:hover{border-color:var(--orange);background:var(--orange-pale)}
-.toggle-track{width:32px;height:18px;border-radius:100px;background:var(--toggle-bg);position:relative;transition:background .3s;border:1.5px solid var(--border);flex-shrink:0}
-.toggle-knob{width:12px;height:12px;border-radius:50%;background:var(--toggle-knob);position:absolute;top:1px;left:1px;transition:transform .3s;box-shadow:0 1px 3px rgba(0,0,0,.2)}
+.theme-toggle:active{transform:scale(0.95)}
+.toggle-track{width:32px;height:18px;border-radius:100px;background:var(--toggle-bg);position:relative;transition:background .4s ease,border-color .4s ease;border:1.5px solid var(--border);flex-shrink:0}
+.toggle-knob{width:12px;height:12px;border-radius:50%;background:var(--toggle-knob);position:absolute;top:1px;left:1px;transition:transform .4s cubic-bezier(0.34,1.56,0.64,1),background .4s ease;box-shadow:0 1px 3px rgba(0,0,0,.2)}
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)